keymap.cson

```diff
@@ -23,3 +23,14 @@
 # This file uses CoffeeScript Object Notation (CSON).
 # If you are unfamiliar with CSON, you can read more about it here:
 # https://github.com/bevry/cson#what-is-cson
+
+# window/pane navigation
+'body, atom-workspace atom-text-editor:not([mini])':
+  'cmd-h': 'window:focus-previous-pane'
+  'cmd-l': 'window:focus-next-pane'
+  'cmd-shift-v': 'pane:split-right'
+  'cmd-shift-s': 'pane:split-down'
+  'cmd-shift-h': 'window:focus-pane-on-left'
+  'cmd-shift-l': 'window:focus-pane-on-right'
```

Comment:
i use cmd-L in sublime. 

split selection into lines is something i'll use as a replacement for stuff i would do in vim w/ visual block mode. Think adding `* ` in front of 10 lines. You'd select the 10 lines, split selection, hit ctrl-a, type `* `

Maybe there's a better way, but i think that's the sublime/multi-cursor approach.

I'd like to avoid remap cascades and keep as many default keybindings as we can.